docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -116,3 +116,13 @@ When you are ready for other people to review your work, perform the following t
 When you add a PR to the repo and assign the `type/docs` label, it will be reviewed by a member of the Docs Squad.
 
 Depending on the size of the PR and the priority of other work, the PR will either be immediately reviewed and merged (minor fixes typically follow this pattern) or the PR will be triaged and placed in the backlog of work or moved into further development.
+
+## Contributing across versions
+
+Editing the main branch of a project affects the content in the "next" directory of the website.
+To edit previous versions including "latest", the most recent release, and other versions, you must backport the changes into the long-lived version branches in the project repository.
+
+Backporting is most easily achieved using the backport vMAJOR.MINOR.x labels on the GitHub pull request.
+
+`grafanabot` automatically creates a backport pull request after the original pull request is merged if the merge commit can be cherry-picked without conflict.
```

Comment:
```suggestion
`grafanabot` automatically creates a backport pull request after the original pull request is merged, if the merge commit can be cherry-picked without a conflict.
```

Reply:
Can we keep "cherry-picked" as it matches with the `git` subcommand (`git cherry-pick`) or perhaps should that dash indicate that I should use "cherry picked"?

Reply:
Yes, definitely. Nice catch.